core/types/transaction_marshalling.go

```diff
@@ -74,6 +74,9 @@ type txJSON struct {
 	PaymasterGas  *hexutil.Uint64 `json:"PaymasterGas,omitempty"`
 	PostOpGas     *hexutil.Uint64 `json:"PostOpGas,omitempty"`
 
+	// RIP 7712 additional transaction field
+	AaNonce *hexutil.Big `json:"aaNonce,omitempty"`
```

Comment:
Is there any reason why the type of the nonce is big.Int instead of uint64? 
If there's no particular reason, it would be nice to make it uint64, which is the same type as the contract section.

Reply:
It should be big.Int, since it should include `key` prefix which is 192 bits. The total length of AaNonce is 256 bits.

Reply:
Oh, yes i got it